### PR TITLE
Redesigned the website of Soundnode

### LINKED ISSUE #791: Promote Ubuntu/Debian packages
Hi @weblancaster!

I am maintaining a [Debian and Ubuntu repository on packagecloud.io](https://packagecloud.io/JonasGroeger/soundnode). This is very nice for Debian and Ubuntu users since the installation and updates are very convenient:

    # Installation
    curl -s https://packagecloud.io/install/repositories/JonasGroeger/soundnode/script.deb.sh | sudo bash
    sudo apt-get update
    sudo apt-get install soundnode

    # Updates come with normal OS updates

Currently, the latest development version is uploaded. However, we could for example have 2 separate repositories: `soundnode` and `soundnode-latest`.

We already have some successfull users of the repository in #775 and #780. My Account on [packagecloud.io](packagecloud.io) is run using the OpenSource package they provide, which means I'm not paying for that (15GB bandwidth and 5GB of storage). Since the package is about 40MB, we can have a total maximum of (15GB / 40MB = ) 375 downloads/month.

I would like to have the instructions on how to install soundnode on Debian/Ubuntu be published / promoted on [soundnodeapp.com](soundnodeapp.com).

How can we arrange that?

### PR DESCRIPTION
**Pull request by ghost**
I redesigned the website of Soundnode. Please let me know what you think about it. Also closes Soundnode/soundnode-app#791.

Screenshot
Screenshot

### CODE REVIEW

**Comment by weblancaster** [comment]
I will be testing this PR over the weekend. It's time for a new website.

**Comment by ghost** [comment]
Yeah, but why aren't you using GitHub Pages?

**Comment by weblancaster** [comment]
I was used to use Heroku for all my side projects.. I might move to GH then.

**Comment by ghost** [comment]
That would make this site a bit easier to maintain as you don't have 2
repo's (1 on GH and 1 on Heroku) and would make the site a bit faster as it
would be served by nginx instead of node.js which isn't meant for static
sites. That would require making a new Soundnode.github.io repo, adding
only the `public` folder of this PR to the root of the new repo and setting
the DNS records using [this guide](https://help.github.com/articles/using-a-custom-domain-with-github-pages/).

File: public/javascript/main.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+function b(){setTimeout(function(){scrollBy(0,-51)},5)}function c(){51<document.getElementById("download").getBoundingClientRect().top?51<document.getElementById("featuredon").getBoundingClientRect().top?(document.getElementsByClassName("download")[0].parentNode.className="",document.getElementsByClassName("featuredon")[0].parentNode.className=""):(document.getElementsByClassName("download")[0].parentNode.className="",document.getElementsByClassName("featuredon")[0].parentNode.className="active"):(document.getElementsByClassName("featuredon")[0].parentNode.className="",document.getElementsByClassName("download")[0].parentNode.className="active")}document.getElementsByClassName("navbar-brand")[0].addEventListener("click",b),document.getElementsByClassName("featuredon")[0].addEventListener("click",b),c(),addEventListener("scroll",c),document.getElementsByClassName("navbar-toggle")[0].addEventListener("click",function(){var e=document.getElementsByClassName("navbar-collapse")[0]
```
Comment (weblancaster) [inline public/javascript/main.js:1]: can you format the code please? not minified(ish)

Comment (ghost) [inline public/javascript/main.js:1]: Why don't you want it minified?

Comment (weblancaster) [inline public/javascript/main.js:1]: Because the build will do that when released.

Comment (ghost) [inline public/javascript/main.js:1]: Which build? By the way I'll do that tomorrow…

**Comment by weblancaster** [comment]
You know what @wb9688 doesn't matter.. let's release this way.

This is great work! thanks a lot for doing this.

**Comment by ghost** [comment]
When will this be online @ soundnodeapp.com?

**Comment by weblancaster** [comment]
~~I'm going to push tonight.. so in a couple of hours.~~

**Comment by weblancaster** [comment]
Actually I enabled auto deploy when merged into master.. should on less than 10mins @wb9688

**Pull request merged**